ee/debug/checkups/checkups.go

```diff
@@ -119,6 +119,7 @@ func checkupsFor(k types.Knapsack, target targetBits) []checkupInt {
 		{&serverDataCheckup{k: k}, doctorSupported | flareSupported | logSupported},
 		{&osqDataCollector{k: k}, doctorSupported | flareSupported},
 		{&osqRestartCheckup{k: k}, doctorSupported | flareSupported},
+		{&uninstallHistoryCheckup{k: k}, doctorSupported | flareSupported},
```

Comment:
```suggestion
		{&uninstallHistoryCheckup{k: k}, flareSupported},
```

You can convince me otherwise, but I don't see there being value in doctor

Reply:
nope this was more just out of habit, thank you!